src/shadowbox/server/main.ts

```diff
@@ -114,35 +115,35 @@ async function main() {
       scrape_interval: '15s',
     },
     scrape_configs: [
-      {job_name: 'prometheus', static_configs: [{targets: [prometheusLocation]}]},
-      {job_name: 'outline-server-main', static_configs: [{targets: [nodeMetricsLocation]}]},
+      { job_name: 'prometheus', static_configs: [{ targets: [prometheusLocation] }] },
```

Comment:
Is this really the style? Looks odd to me. Maybe because it's multi line...

Reply:
I guess? I tend to not bother arguing with autoformatters, and this isn't that heinous.